health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/AppFeatures.js

```
@@ -0,0 +1,97 @@
+          options={toggleOptions}
+          optionsKey="code"
+          selectedOption={code}
+          style={{}}
+          t={function noRefCheck() {}}
+          type="toggle"
+          value=""
+        />
+      </div>
+      <div className="modules-container">
+        {selectedFeatures?.map((module, index) => (
+          <Card className={"module-card"}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)